earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -200,7 +147,12 @@ ()
         $formalcachedinputarg
         $formalExtraUpdateArg)
 {
-    Mutable${name}AssetD asset = FindMake(ref_ $forwardtypearg,
+EOF
+
+print $fh <<EOF;
+
+    Mutable${name}AssetD asset = AssetFactory::FindMake<Mutable${name}AssetD,$config>
```

Comment:
Minor: for simplicity, everywhere FindMake is called I think you can leave out the second template parameter and just call
```
AssetFactory::FindMake<Mutable${name}AssetD>( /* args ... */ )
```